### ISSUE TITLE
I can't see the water height data in Windows

### ISSUE THREAD

**Issue by huangrh99** [P3TE/Autonomous-Surface-Vehicle-Simulator#3]
When I ran this simulator in Linux, I can see the boat shaking with wave, and the water height data. But I can't see those when I ran in Windows

**Comment by P3TE** [comment]
I see, this is not very well documented, but the settings that you can change whilst the simulator is running don't actually do anything anymore. I recently changed the way that the simulated water is calculated. It now has more parameters so you can have a more realistic water. However you can only change the parameters in code.
If you wish to have dynamic waves, you have to change parameters within RenderedWater.java. Sea state  (based off https://en.wikipedia.org/wiki/Sea_state) for i up to 5 is modeled with the parameters:
seaStateAValues, seaStateWValues, seaStateThetaValues, seaStateKValues
These values currently start on line 294.

Basically, try setting the first seaStateAValues to 0.3 and you should be able to see a dynamic ocean. 
Also make sure that you select complex water (Press on simple water in the in-simulator menus)

**Comment by huangrh99** [comment]
Thanks, it works.
